data/os/Ubuntu/18.04.yaml

```diff
@@ -0,0 +1,3 @@
+---
+postgresql::repo::baseurl: https://apt-archive.postgresql.org/pub/repos/apt/
+postgresql::repo::release: "%{facts.os.distro.codename}-pgdg-archive"
```

Comment:
Having archive repos is much cleaner than using a different distro. I much prefer this.